lib.nix

```diff
@@ -3,7 +3,11 @@ let
   fetchNixPkgs = let try = builtins.tryEval <cardano_pkgs>;
     in if try.success
     then builtins.trace "using host <cardano_pkgs>" try.value
-    else import ./fetch-nixpkgs.nix;
+    else import ./nix/fetch-nixpkgs.nix;
```

Comment:
`fetch-nixpkgs.nix` is only a hack to allow `nix-build` in the buildkite hooks
the nix scripts should always be using `fetchNixpkgs.nix`

Reply:
@cleverca22, is this distinction documented anywhere?

It's somewhat confusing there are two competing approaches that are pronounced the same way. Renaming one would help.